lensless/recon.py

```diff
@@ -332,9 +338,41 @@ def set_data(self, data):
         else:
             self._data = data
 
+    def _set_initial_estimate(self, image_est):
+        """
+        Set initial estimate of image, e.g. to warm-start algorithm.
+
+        Note that reset() should be called after setting the initial estimate
```

Comment:
@YohannPerron I've added this note that reset() should be called if someone calls this function. It's also a hidden method (starting with `_`) to dissuade people from using this...